toolbox/cijoe-pkg-xnvme/testplans/xnvme-base-macos.plan

```diff
@@ -0,0 +1,32 @@
+---
+descr: Verify base xNVMe functionality using the macOS backend
+descr_long: |
+  Exercises 
+  * library-information
+  * device enumeration
+  * admin-commands idfy-ns, idfy-ctrlr, get-log, get/set feature
+  * queue-init/teardown of io_uring/libaio/posix
+  Requires
+  * XNVME_URI points to an NVMe namespace e.g. /dev/nvme0n1
+evars:
+  NVME_NSTYPE: "lblk"
+  DEV_TYPE: "nvme"
+  XNVME_BE: "macos"
+  XNVME_ADMIN: "macos"
+  XNVME_URI: "/dev/disk4"
```

Comment:
The ``XNVME_URI`` be removed from here and instead adjust the construction of ``XNVME_URI`` in ``toolsbox/cijoe-pkg-xnvme/envs/refenv-xnvme.sh`` and ``toolsbox/cijoe-pkg-xnvme/envs/refenv-xnvme.sh`.